test/e2e-common.sh

```diff
@@ -288,7 +288,6 @@ function install() {
     > "${ytt_result}" \
     || fail_test "failed to create deployment configuration"
 
-
```

Comment:
Total nit: Can we drop this unnecessary change? LGTM otherwise.

Reply:
Sure! updated.